peerconnection.go

```diff
@@ -1288,17 +1359,67 @@ func (pc *PeerConnection) startRTPReceivers(incomingTracks []trackDetails, curre
 			unhandledTracks = append(unhandledTracks, filteredTracks[i])
```

Comment:
@boks1971 Is this a problem?

Reply:
Forgot to clean this up. This is used if `remoteIsPlanB`. I could not get `PlanB` test to work properly by splitting the setup and start of receivers. I will try it again today and post exactly what the issue is.